_posts/2015-08-07-MakingMusicFun.md

```diff
@@ -1,54 +1,27 @@
-change how we perceive our musical experiences. This can be especially prevalent
-in the music that we listen to, let alone what we play on our instruments.
-Perhaps after going through a break-up, something about Sam Hunt’s “Take Your
-Time” really hits home. Or suddenly Eric Paslay and Little Big Town become
-favorites that you may not have known existed before. Often when something in
-your life changes, the way you perceive the lyrics and sounds of a genre
-changes. Perhaps this leads you to find a new love for a new genre. It could
-also reignite your excitement with an old genre or artist you listened to in the
-past. Maybe you end up exploring something completely new to you! We have to
-remind ourselves that our taste in music changes at different points in our
-lives and, along with it, our interest, excitement, and emotional attachment
-to playing certain pieces might as well. It is important to consider whether
-your interest in listening to a particular artist or genre has changed. If it
-has, do you need to modify what you are playing to maintain your interest in
-practicing your instrument?
-
-Sometimes because of a loss of interest or desire the time and availability you
-once set aside for practicing disappears. Maybe life has become busier or
-practicing just stopped seeming as important as it once was. When these things
-happen, it is important to find a reason to practice or learn something new. I
-try to ensure that my students and I are always working towards a performance.
-I like to have quarterly recitals or get them playing in seniors' homes monthly.
-For myself, I like to learn a new piece or revamp an old one for each
-performance. If you think that you need to find a reason to practice or learn
-new material, you can schedule yourself a performance or even a jam session with
-friends. Make the performance or jam part of your routine to keep yourself
-practicing on a consistent basis.
-
-While I encourage working towards performances with my students I also try to
-ensure that they have the opportunity to define what they want to learn and then
-learn some of the things that they discover they like. I encourage them to
-explore and to try composing their own pieces. Sometimes falling into a rut
-happens because we feel like the music we play is too rehearsed. Sometimes we
-realize the challenge and "unknown" that was originally there when we began
-learning is gone. If music is not exciting for you anymore then we must find a
-way to make it exciting. Try writing a harmony to a piece, learning more about
-music theory or music history, expanding your musical knowledge and capability,
-or even taking up a new instrument!
-
-The bottom line is: when things seem to be getting mundane we must remember what
-inspired us in the first place and try to recapture that musical magic in new
-and exciting ways.
+<p>Stuck in a rut? Do you want to make music learning fun again?</p>
```

Comment:
Whoa, @robgillan this is extremely wrong. You do not put raw html in markdown files.